storage/kv.go

```diff
@@ -46,3 +59,32 @@ type KV interface {
 	Restore() error
 	Close() error
 }
+
+// Watcher uses to watch on key-value store.
```

Comment:
Watcher watches on the KV. It will be notified if there is an event happened on the watched key or prefix.